implementation/pom.xml

```diff
@@ -23,6 +23,10 @@
             <artifactId>smallrye-common-annotation</artifactId>
             <version>${smallrye-common-annotation.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.jctools</groupId>
+            <artifactId>jctools-core</artifactId>
```

Comment:
Did you check with Paul about prodictization?

Reply:
Yes it's a non-issue